core/src/main/resources/error/error-classes.json

```diff
@@ -1580,6 +1580,11 @@
       "Not allowed to implement multiple UDF interfaces, UDF class <className>."
     ]
   },
+  "NAMED_ARGUMENTS_SUPPORT_DISABLED" : {
```

Comment:
Sounds like it can be enabled. If so, could you describe how in the error message.

> ... are not supported here

If it is just not supported, could you add a sub-class to `UNSUPPORTED_FEATURE`